renderer/components/Titlebar.tsx

```diff
@@ -0,0 +1,69 @@
+// import { BrowserWindow } from 'electron'
+import { Show } from 'solid-js';
+// eslint-disable-next-line import/no-unresolved
+import { JSX } from 'solid-js/jsx-runtime';
```

Comment:
```suggestion
import type { JSX } from 'solid-js/jsx-runtime';
```
타입 import eslint가 뭐라하면 import type으로 해주심 됩니다